ui/src/views/Home.tsx

```diff
@@ -96,14 +95,6 @@ function Home() {
               >
                 Add Pool
               </Link>
-              <Link
-                component={NavLink}
-                to="/pools"
-                color="inherit"
-                className={classes.link}
-              >
-                Pools
-              </Link>
```

Comment:
I removed the pools page to avoid query contract events from the full node, I think we can add it back when [the explorer-backend supports query subcontract ids](https://github.com/alephium/explorer-backend/pull/434).